lib/proofer/vendor/mock.rb

```diff
@@ -29,7 +29,7 @@ def submit_financials(financials, session_id = nil)
       end
 
       def submit_phone(phone_number, session_id = nil)
-        if phone_number == '(555) 555-5555'
+        if phone_number =~ /\A(\+1 )?\(555\) 555\-5555\z/
```

Comment:
In the spirit of:

> Some people, when confronted with a problem, think "I know, I'll use regular expressions." Now they have two problems.

Does the formatting matter? Would it be simpler to strip non-digit characters for checking?

```ruby
digits_only = phone_number.gsub(/\D/, '')
if phone_number == '5555555555' || phone_number == '15555555555'
```